### PR TITLE
linuxraw: Add Nintendo Switch Pro Controller

### PR DESCRIPTION
**Pull request by davidhedlund**

### CODE REVIEW

File: linuxraw/Nintendo Co., Ltd. Pro Controller.cfg [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Screenshot button (circle) is unbound in this autoconf, but has an ID of 4
+
+input_driver = "linuxraw"
+# The files "Nintendo Co., Ltd. Pro Controller.cfg" and "Nintendo Switch Pro Controller.cfg" should have identical content except for the input_device variable. This difference allows thelinuxraw driver to recognize the controller on various Linux kernel versions.
+input_device = "Nintendo Co., Ltd. Pro Controller"
```
Comment (RobLoach) [inline linuxraw/Nintendo Co., Ltd. Pro Controller.cfg:5]: Mind adding the vendor and product ids?

**Review by RobLoach — CHANGES_REQUESTED** [review]

File: linuxraw/Nintendo Co., Ltd. Pro Controller.cfg (hunk 1 above)
Comment (davidhedlund) [reply linuxraw/Nintendo Co., Ltd. Pro Controller.cfg:5]: The linuxraw driver does not utilize the Vendor ID and Product ID. This is thoroughly explained in the documentation available at https://docs.libretro.com/guides/controller-autoconfiguration/.

Please review the entire guide (https://docs.libretro.com/guides/controller-autoconfiguration/), as I have [invested considerable time in enhancing it](https://github.com/libretro/docs/commits/master/docs/guides/controller-autoconfiguration.md?author=davidhedlund).

**Comment by davidhedlund** [comment]
I will create a group of pull requests that include both HID and non-HID auto-configurations for linuxraw.

File: linuxraw/Nintendo Co., Ltd. Pro Controller.cfg (hunk 1 above)
Comment (davidhedlund) [reply linuxraw/Nintendo Co., Ltd. Pro Controller.cfg]: They are not generated or utilized by this controller driver

**Pull request merged**